cluster/gce/gci/mounter/mounter

```diff
@@ -17,4 +17,20 @@
 set -e
 set -o pipefail
 
-sudo /bin/mount "$@"
\ No newline at end of file
+MOUNTER_DOCKER_IMAGE=gcr.io/google_containers/gci-mounter
+MOUNTER_DOCKER_VERSION=v1
+MOUNTER_USER=root
+RKT_BINARY=/home/kubernetes/bin/rkt
```

Comment:
This won't work with the node e2e tests yet for a few reasons:
-  `/home/kubernetes/...` isn't a thing in node e2e
- node e2e GCI images don't have rkt installed anyway
- node e2e tests won't run the `cluster/gce/gci/configure.sh` script when they spin up a testing node, so https://github.com/kubernetes/kubernetes/pull/35321 won't have any effect on them.

Probably the easiest hack to fix this is to have the node test runner just create a `/home/kubernetes/` and drop `rkt` in there.